### DISCUSSION TITLE
Template repo for starting out a simple Roda application

### DISCUSSION THREAD

**Discussion by nini1294** [jeremyevans/roda#272] [General]
Hi all,

I found that a lot of times I want to create a simple Roda project and start out with a scaffold/template, so I created a git repo that I can clone and have running without too much fuss. I used the Roda [conventions](http://roda.jeremyevans.net/rdoc/files/doc/conventions_rdoc.html#label-Small+Applications) page as a starting point and added [rack-unreloader](https://github.com/jeremyevans/rack-unreloader) and a few additional rake tasks on top.

https://gitlab.com/nini1294/roda-template-simple

Please check it out, suggestions, bug reports and pull requests are welcome!

**Comment by jeremyevans** [comment]
Thanks for sharing.  If you are interested, can you send a pull request that modifies `www/pages/documentation.erb` to add a link to roda-template-simple (in the external projects section)?

**Reply by nini1294** [reply]
Sure, just created this PR https://github.com/jeremyevans/roda/pull/273
